### PR TITLE
fix: temporary remove functions with sparse matrices

### PR DESCRIPTION
**Pull request by Antonov548**
https://github.com/igraph/rigraph/issues/1006

### CODE REVIEW

File: R/aaa-auto.R [hunk 1]
```diff
@@ -3989,6 +3938,18 @@ stochastic_imitation_impl <- function(graph, vid, algo, quantities, strategies,
   res
 }
 
+has_attribute_table_impl <- function() {
```
Comment (krlmlr) [inline R/aaa-auto.R:3941]: Can you please remove the two R functions too?

**Review by krlmlr — APPROVED** [review]
Thanks!

**Comment by szhorvat** [comment]
This could have been done by adding empty entries to `functions-R.yaml`. That way we don't create conflicts with auto-generation. For example:

```
igraph_sparse_weighted_adjacency:

```

**Comment by krlmlr** [comment]
Moving forward, we could add comments if we add intentional drift so that the next person that updates the generated code sees where the drift comes from.

**Pull request merged**